plugins/modules/content_template.py

```diff
@@ -37,7 +37,6 @@
         description:
         - The name of the VM to be used to create template.
         type: str
-        required: true
```

Comment:
I think it's worth mentioning that `vm_name` is required when the state is `present`. Maybe not here, but in the description.